src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNameKind.cs

```diff
@@ -35,6 +35,8 @@ internal enum GeneratedNameKind
         AsyncBuilderField = 't',
         DynamicCallSiteContainerType = 'o',
         DynamicCallSiteField = 'p',
+        MethodGroupConversionCacheFrame = 'q',
+        MethodGroupConversionCacheDelegateField = 'r',
```

Comment:
As far as I can tell, 0-9 and a-u are spoken for.  I'll file a bug to add placeholders to this enum.